essentials-module-dynmap/src/main/java/io/github/nyliummc/essentials/mixin/dynmap/DynmapCoreMixin.java

```diff
@@ -0,0 +1,25 @@
+package io.github.nyliummc.essentials.mixin.dynmap;
+
+import io.github.nyliummc.essentials.api.EssentialsMod;
+import net.fabricmc.loader.api.FabricLoader;
+import org.apache.commons.io.IOUtils;
+import org.dynmap.DynmapCore;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+import java.io.*;
+
+@Mixin(DynmapCore.class)
+class DynmapCoreMixin {
+    @Redirect(method="initConfiguration", at=@At(value="NEW", target="java.io.File", ordinal=0))
+    File configFile() throws IOException {
+        File f = new File(FabricLoader.getInstance().getConfigDirectory().getCanonicalPath() + "/essentials-dynmap.yaml");
```

Comment:
Should we making an essentials folder?

We seem to be annexing most of the root server folder.

Reply:
Good idea, but out of scope for this PR I think